### PR TITLE
RHCLOUD-33925 Button links in stage point to wrong env.

### PR DESCRIPTION
**Pull request by aferd**
### Description

Button links in stage currently point to Prof env, switched it to use relative paths.
Added ouiaId's to all the buttons.

[RHCLOUD-33925](https://issues.redhat.com/browse/RHCLOUD-33925)

---

#### After:

---

### Checklist ☑️
- [ ] PR only fixes one issue or story 
- [ ] Change reviewed for extraneous code 
- [ ] UI best practices adhered to 
- [ ] Commits squashed and meaningfully named 
- [ ] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

File: src/components/widgets/explore-capabilities.tsx [hunk 1]
```diff
@@ -37,7 +39,10 @@ const ExploreCapabilities: React.FunctionComponent = () => {
       title: 'Get started with Red Hat OpenShift Service on AWS (ROSA)',
       body: 'Quickly build, deploy, and scale applications with our fully-managed turnkey application platform.',
       buttonName: 'Try ROSA',
-      url: 'https://console.redhat.com/openshift/overview/rosa',
+      ouiaId: 'try-rosa-button',
+      url: isProd()
+        ? 'https://console.redhat.com/openshift/overview/rosa'
+        : 'https://console.stage.redhat.com/openshift/overview/rosa',
```
Comment (aneelac22) [inline src/components/widgets/explore-capabilities.tsx:45]: You should be able to use relative path instead of using conditionals

**Review by aneelac22 — APPROVED** [review]

**Comment by catastrophe-brandon** [comment]
Looks like the PR build failed due to a test failure in `widgets/rhel.cy.ts`

**Review by fhlavac — APPROVED** [review]

**Comment by catastrophe-brandon** [comment]
@aferd we'll need to incorporate the tests that were just merged to master in order to get the PR build to pass. It should be an easy merge or rebase to your branch, but lmk if you have issues

**Pull request merged**